newsfragments/validate_files.py

```diff
@@ -28,4 +28,5 @@
 
     full_extension = "".join(fragment_file.suffixes)
     if full_extension not in ALLOWED_EXTENSIONS:
-        raise Exception(f"Unexpected file: {fragment_file}")
\ No newline at end of file
+        raise Exception(f"Unexpected file: {fragment_file}")
+    
```

Comment:
I think you added one newline too much here (hence the little pointer is still there)